### PR TITLE
[chore]: wire up static directory, scaffold out css for various views

### PR DESCRIPTION
**Pull request by mcMickJuice**
In anticipation of styling the app, I created css files for each view. I also wired up the root `public` folder to be a static directory to follow patterns I've used in express apps.  Css files can be referenced in templates with `/css` as the root. When/if we add javascript files, we'll add a `js` directory to the `public` directory and reference scripts with `/js` as the root.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ app.engine(".hbs", engine({ extname: ".hbs" }));
 app.set("view engine", ".hbs");
 app.set("views", path.resolve(__dirname, "./views"));
 
+app.use(express.static("public"));
```
Comment (mcMickJuice) [inline src/server.js:16]: All static asset requests will be looked up in the `public` directory, located in the root of the project.

File: src/views/search.hbs [hunk 2]
```diff
@@ -1 +1,2 @@
+<link rel="stylesheet" type="text/css" href="/css/search.css" />
```
Comment (mcMickJuice) [inline src/views/search.hbs:1]: Maybe all search related styling will live in `presentations.css` and this `search.css` file isn't necessary...?

**Review by susannnnxiao — APPROVED** [review]
looks good to me!

File: public/css/styles.css
Comment (bossley9) [inline public/css/styles.css:1]: Oh, you can rename this to something more specific 😅 maybe `animations.css`? Up to you

File: src/server.js (hunk 1 above)
Comment (bossley9) [reply src/server.js:16]: I added a similar line below on line 40ish, can probably combine the two

**Review by bossley9 — APPROVED** [review]

File: src/server.js (hunk 1 above)
Comment (mcMickJuice) [reply src/server.js:17]: [Done here](https://github.com/monicaleep/member-fe-microsite/pull/17/commits/1e87ef1d7af6a2734d8cdc794a77a4c85e2e6c04)

**Pull request merged**